sqflite/lib/src/types/types.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:jaguar_query/jaguar_query.dart';
+
+class A extends Expression implements ToDialect {
+  final List<Expression> items;
+
+  A(this.items);
```

Comment:
What does `A` mean ?? from the `toDialect` I can guess it's for Array but would be nice to have a more meaningful name for this class :)

Reply:
I just made the same class as in the postgres/lib/src/types/types.dart.
For easy maintenance I think this we should keep the code the most similar since this is the adapter.